files/bsd/netbsd-rc.local.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Save to /etc/rc.local, to be ran on startup
```

Comment:
The scripts for net and openbsd are quite similar. What would it take to share them?